_pages/rfc-process.md

```diff
@@ -25,31 +25,42 @@ An RFC can extend, modify, or alter any part of the WeAllJS community policy, wh
 
 ## How do I create an RFC?
 
-Easy: Just join the #meta channel and create a new Slack Post by clicking on the + next to the message box. Prefix whatever title you're using for your RFC with RFC -, and then share it with #meta once you've drafted the initial document.
-
-There's currently no set structure to what these documents should look like, but generally an initial introductory section followed by a series of headers is a pretty good way to write readable documentation.
-
-If you're unsure if the RFC topic is relevant or appropriate, feel free to ask in #meta, gauge interest, or even use /admin to talk to an administrator about what you're thinking of proposing.
+* https://github.com/wealljs/wealljs.github.io
+* Create a PR with a new RFC document:
+  * Must be in `_pages`.
+  * Must be prefixed by `rfc-` and be a `.md` file.
+  * Must use the standard page header (refer to other RFCs)
+  * The first line should be `# RFC - <title of RFC>`
+* If the RFC involves existing or new documents on the website, include those changes in the same PR for review. They will be merged alongside the RFC when accepted.
+* Tag at least one member of the [Admin Team](https://github.com/orgs/WeAllJS/teams/admins) for review on the PR.
+* Join `#meta` in the WeAllJS Slack and announce the PR.
+* Pin the link to the PR to `#meta`.
 
 <div class="bordered-divider bordered-bottom"></div>
 
 ## How does review work?
 
-All comments on a particular RFC should be done directly on the document: Slack posts have a small speech bubble on the top right of the document window that can be used to add new comments to the document. This helps preserve any conversation that happened, save questions, etc, which will be protected from the Slack's 10k message limit. As much discussion as possible should be handled in the general context of #meta or private messages, or even private channels, but the final document must be available and visible to all members, and be shared in #meta.
+All comments on a particular RFC should be done directly on the PR: An RFC will not be accepted until reviewers are satisfied with changes. Admins hold ultimate authority over whether or not to accept an RFC, if there's ever conflict about it.
+
+Side discussion on the RFC may happen anywhere, though external discussion is better off in the `#meta` channel for better visibility. Please record any relevant decisions as comments on the PR, when they happen in external discussions.
```

Comment:
Suggestion for clarity: Discussion on the RFC may happen outside the PR, but is best kept in the WeAllJS slack `#meta` channel for better community visibility.